### PR TITLE
fix: adjust label for coinbase prime api key form

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by yabirgb — APPROVED** [review]
looks okey to me

imagen

File: frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue [hunk 1]
```diff
@@ -185,8 +197,19 @@ function onExchangeChange(exchange?: string) {
...
+  if (location === 'coinbase') {
+    return markRaw(defineAsyncComponent(() => import('@/components/settings/api-keys/exchange/CoinbaseKeyForm.vue')));
+  }
+
+  if (location === 'coinbaseprime') {
+    return markRaw(defineAsyncComponent(() => import('@/components/settings/api-keys/exchange/CoinbaseProKeyForm.vue')));
+  }
+
+  return markRaw(defineAsyncComponent(() => import('@/components/settings/api-keys/exchange/RegularKeyForm.vue')));
+});
```
Comment (kelsos) [inline frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue:212]: Do we really need 3 components and all this complicated logic? can't we do a single one that takes the location?

Comment (lukicenturi) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue:212]: Some of them have a different order of input and also change the label of the input.
 I just thought this way was more readable.

Comment (kelsos) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue:212]: I wonder if we could also do the order in a dynamic way as part of the component/config?

Comment (kelsos) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue:212]: let me think a bit about it.

Comment (kelsos) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue:212]: Would it work if we use defineSlots to define the slots and payload and also create a structure that contains 

```typescript
type SlotProps = {label: string, hint: string}
const slots = defineSlots<{
 apiKey(props: SlotProps): any,
 apiSecret(props: SlotProps): any,
 passphrase(): any
}>();

const data = computed(() => [{
 slot: slots.apiKeys,
 bindings: { hint, label },
}, ...]
```

And in the template we do something with the syntax
```vue
  <template v-for="(_, name) in slots" v-slot:[name]="slotData"><slot :name="name" v-bind="slotData" /></template>
 ```
 
The how of the the template probably needs a bit of figuring out but this way we can keep the single component and also provide custom ordering of the slots.

Comment (lukicenturi) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue]: I pushed again, not exactly like this, but it's 1 component now, can you check again? @kelsos

Comment (kelsos) [reply frontend/app/src/components/settings/api-keys/exchange/ExchangeKeysForm.vue]: The implementation is basically what I had in mind.

Thank you :1st_place_medal:

**Pull request merged**